src/vue/pages/Sales.vue

```diff
@@ -71,7 +71,8 @@
       </drawer>
     </template>
 
-    <template v-if="getModule().canRenderSubmodule(CreateSaleFormModule)">
+    <!-- eslint-disable-next-line max-len -->
+    <template v-if="getModule().canRenderSubmodule(CreateSaleFormModuleSimplified)">
```

Comment:
the page should support both versions of `create-form`, do not replace but add the new forms